services/apps/alcs/src/alcs/admin/board-management/board-management.controller.ts

```diff
@@ -49,7 +49,7 @@ export class BoardManagementController {
   @UserRoles(AUTH_ROLE.ADMIN)
   async canDelete(@Param('code') code: string) {
     //If it's the vetting board
```

Comment:
nitpick the comment does not match the code

Reply:
Fixed